website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,191 @@
+
+## Breaking Changes
+
+### Deprecated Component Removals
+
+The following packages have been removed from React Native in 0.72. Please migrate to the recommended community package:
+
+- [Slider](https://reactnative.dev/docs/next/slider) is superseded by [@react-native-community/slider](https://github.com/callstack/react-native-slider/tree/main/package)
+- [DatePickerIOS](https://reactnative.dev/docs/next/datepickerios) is superseded by [@react-native-community/datetimepicker](https://github.com/react-native-datetimepicker/datetimepicker)
+- [ProgressViewIOS](https://reactnative.dev/docs/next/progressviewios) is superseded by [@react-native-community/progress-view](https://github.com/react-native-progress-view/progress-view)
+
+### Package Renames
+
+All packages published from the repository now live under `react-native/packages`, and are published under the [@react-native npm scope](https://www.npmjs.com/search?q=%40react-native) to ensure clear ownership.
+
+There are no changes to the [react-native](https://www.npmjs.com/package/react-native) package.
+
+Here are all the packages renamed:
+
+- `@react-native-community/eslint-config` to `@react-native/eslint-config`
+- `@react-native-community/eslint-plugin` to `@react-native/eslint-plugin`
+- `@react-native/polyfills` to `@react-native/js-polyfills`
+- `@react-native/normalize-color` to `@react-native/normalize-colors`
+- `@react-native/assets` to `@react-native/assets-registry`
+- `react-native-codegen` to `@react-native/codegen`
+- `react-native-gradle-plugin` to `@react-native/gradle-plugin`
+
+This change will not affect you if you have no direct dependency on a renamed package. Otherwise, when upgrading to React Native 0.72, update any renamed dependency to its ~0.72 version.
+
+You can read background on these changes [here](https://github.com/react-native-community/discussions-and-proposals/pull/480).
+
+## Acknowledgements
+
+A lot of this release came from the direct feedback from the community. From reports on [noisy redboxes](https://twitter.com/baconbrix/status/1623039650775371792), [bugs with Package Exports](https://github.com/facebook/metro/issues/965), [benchmarks for the New Architecture](https://github.com/reactwg/react-native-new-architecture/discussions/85) — all of it is valuable to hear and we appreciate the time it takes to share feedback.
+
+In addition, thank you to the contributors to this release! 66 contributors submitted over [1100 commits](https://github.com/facebook/react-native/compare/v0.71.8...v0.72.0)! Thank you all for your hard work!
+
+## Upgrade to 0.72.0 now!
+
+Check out the list of needed changes in the [upgrade-helper](https://react-native-community.github.io/upgrade-helper/), or read the [upgrade documentation](https://reactnative.dev/docs/upgrading) for how to update your existing project, or create a new project with `npx react-native@latest init MyProject`.
+
+React Native version 0.72 will be supported in Expo SDK version 49.
```

Comment:
```suggestion
If your project is based on Expo there is a good news too, React Native version 0.72 will be supported in upcoming Expo SDK 49 release.
```

Expo SDK phrasing nit, not sure tho if this style fits writing style of the whole article. Feel free to change or even ignore this suggestion if this feels off.